code/modules/mob/living/carbon/brain/emote.dm

```diff
@@ -49,9 +49,17 @@
 			src << "You whistle."
 			message = "<B>[src]</B> whistles."
 			m_type = 2
-		if ("beep")
-			src << "You beep."
+		if("beep")
 			message = "<B>[src]</B> beeps."
+			playsound(src.loc, 'sound/machines/twobeep.ogg', 50, 0)

```

Comment:
This appears to be missing feedback messages to the user (src)? Even with sound ques, a text response is preferred for those who play without sound for whatever reason, surely?